src/installation/guides/fde.md

```diff
@@ -1,132 +1,131 @@
 # Full Disk Encryption
 
-**Warning**: Your drive's block device and other information may be different,
-so make sure it is correct.
+**Warning**: This guide uses `/dev/vda` as the example device name, Your drive's block device and other information may be different, so make sure it is correct.
 
 ## Partitioning
 
 Boot a live image and login.
 
-Create a single physical partition on the disk using
-[cfdisk](https://man.voidlinux.org/cfdisk), marking it as bootable. For an MBR
-system, the partition layout should look like the following.
+For a BIOS/MBR system, Create a single physical partition on the disk using
+[cfdisk](https://man.voidlinux.org/cfdisk) with a `dos` label type, Mark it as bootable.  
+Now the partition layout should look similar to the following.
```

Comment:
Really, We are going to nit pick the wording change i chose?

I chose "similar to" instead of "like" because the readers partition layout may be different so as long as its **similar to** the example it should be fine.

Why drop BIOS/MBR? This guide doesnt cover BIOS/GPT so i thought the guide should specific.

Reply:
Yes, that's the point of a review. The point is to avoid making stylistic changes to an existing guide when there is no functional purpose for them.

BIOS/MBR is unnecessary because you already instruct people to make a 'dos' label, which is the nomenclature for MBR in the partition program.